docs/index.rst

```diff
@@ -1,17 +1,41 @@
-Welcome to gufe's documentation!
-================================
+gufe
+====
+
+**gufe** is Open Free Energy's framework for defining object models used in alchemical free energy calculations.
+It is intended for use by free energy method developers, and direct access by end-users should be avoided.
+
+If you are looking to perform free energy calculations, see `openfe <https://docs.openfree.energy/>`_.
+
+Resources
+---------
+
+* github repository: https://github.com/OpenFreeEnergy/gufe
+* issue tracker: https://github.com/OpenFreeEnergy/gufe/issues
+* openfe discussions board: https://github.com/OpenFreeEnergy/openfe/discussions
+
+Related Projects
+----------------
+
+.. add links
+
+Open Free Energy Ecosystem:
+
+* `openfe <https://docs.openfree.energy/>`_: The OpenFE ecosystem's central toolkit for performing free energy calculations.
+* `lomap <https://lomap.openfree.energy/>`_: Atom mapping tools from the `Mobley Lab <https://mobleylab.org/>`_
+* `kartograf <https://kartograf.openfree.energy/>`_: Atom mapping tools focusing on 3D geometries.
+* `konnektor <https://konnektor.openfree.energy/>`_: Network generation tools designed with alchemical networks in mind.
+* `cinnabar <https://cinnabar.openfree.energy/>`_: Tools for the analysis of free energy network results.
+
```

Comment:
Add: `alchemiscale <https://alchemiscale.org/>`_: A distributed execution system for alchemical networks.